packages/eslint-config-schibsted-chai/yarn.lock

```diff
@@ -12,15 +12,15 @@ acorn@^3.0.4:
   version "3.3.0"
```

Comment:
delete nested yarn lockfiles

Reply:
I would have expected that to be cleaned up automatically.

Done.